### ISSUE TITLE
libpff doesn't build with system libfmapi

### ISSUE THREAD

**Issue by blshkv** [libyal/libpff#43]
The configure script detects it:   
 libfmapi support:                         **yes**

However, it fails later with the following:

```
/bin/sh ../libtool  --tag=CC   --mode=compile x86_64-pc-linux-gnu-gcc -DHAVE_CONFIG_H -I. -I../common  -I../include -I../common    -march=native -O2 -pipe -Wall -c -o libpff_name_to_id_map.lo libpff_name_to_id_map.c
libtool: compile:  x86_64-pc-linux-gnu-gcc -DHAVE_CONFIG_H -I. -I../common -I../include -I../common -march=native -O2 -pipe -Wall -c libpff_name_to_id_map.c  -fPIC -DPIC -o .libs/libpff_name_to_id_map.o
In file included from libpff_name_to_id_map.c:24:0:
libpff_name_to_id_map.c: In function ‘libpff_name_to_id_map_entry_read’:
libpff_name_to_id_map.c:687:8: error: ‘libfmapi_guid_public_strings’ undeclared (first use in this function)
        libfmapi_guid_public_strings,
        ^
../common/memory.h:109:41: note: in definition of macro ‘memory_copy’
  memcpy( (void *) destination, (void *) source, count )
                                         ^
libpff_name_to_id_map.c:687:8: note: each undeclared identifier is reported only once for each function it appears in
        libfmapi_guid_public_strings,
        ^
../common/memory.h:109:41: note: in definition of macro ‘memory_copy’
  memcpy( (void *) destination, (void *) source, count )
                                         ^
make[1]: *** [Makefile:902: libpff_name_to_id_map.lo] Error 1

```
The full log is here:
https://gist.github.com/blshkv/c560730f79e8773122d2d21ca716b39b

**Comment by joachimmetz** [comment]
thx, I'll have a look as soon as time permits

**Comment by joachimmetz** [comment]
Duplicate of https://github.com/libyal/libpff/issues/42
